### PR TITLE
docs: update cli eslint plugin configuration

### PR DESCRIPTION
**Pull request by jeneser**
Closes: #5568 

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [x] Docs
- [ ] Underlying tools
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

**Other information:**

### CODE REVIEW

**Comment by jeneser** [comment]
yeah, no problem.

File: docs/core-plugins/eslint.md [hunk 1]
```diff
@@ -23,7 +23,11 @@
 
 ## Configuration
 
-ESLint can be configured via `.eslintrc` or the `eslintConfig` field in `package.json`.
+ESLint can be configured via `.eslintrc` or the `eslintConfig` field in `package.json`. See the [ESLint configuration docs](https://eslint.org/docs/user-guide/configuring) for more detail.
+
+::: tip
+There some options are under the section of [`vue.config.js`](https://cli.vuejs.org/config/#vue-config-js). It is respected only when `@vue/cli-plugin-eslint` is installed.
```
Comment (sodatea) [inline docs/core-plugins/eslint.md:29]:
```suggestion
The following option is under the section of [`vue.config.js`](https://cli.vuejs.org/config/#vue-config-js). It is respected only when `@vue/cli-plugin-eslint` is installed.
```

File: packages/@vue/cli-plugin-eslint/README.md [hunk 2]
```diff
@@ -23,7 +23,11 @@
 
 ## Configuration
 
-ESLint can be configured via `.eslintrc` or the `eslintConfig` field in `package.json`.
+ESLint can be configured via `.eslintrc` or the `eslintConfig` field in `package.json`. See the [ESLint configuration docs](https://eslint.org/docs/user-guide/configuring) for more detail.
+
+::: tip
+There some options are under the section of [`vue.config.js`](https://cli.vuejs.org/config/#vue-config-js). It is respected only when `@vue/cli-plugin-eslint` is installed.
```
Comment (sodatea) [inline packages/@vue/cli-plugin-eslint/README.md:29]:
```suggestion
The following option is under the section of [`vue.config.js`](https://cli.vuejs.org/config/#vue-config-js). It is respected only when `@vue/cli-plugin-eslint` is installed.
```

**Comment by jeneser** [comment]
:thumbsup:
I really appreciate your help!

**Pull request merged**